### PR TITLE
SYNC-1097 v2.1 counts

### PR DESCRIPTION
**Pull request by johnhuangclever**
Add counts to v2.1

### CODE REVIEW

File: full-v2.yml [hunk 1]
```diff
@@ -882,6 +897,9 @@ paths:
       - in: query
         name: ending_before
         type: string
+      - in: query
+        name: count
+        type: integer
```
Comment (meliaj) [inline full-v2.yml]: I noticed in the sis-api PR that we can't use summaries for district admins, but can't we still count them?

Comment (johnhuangclever) [reply full-v2.yml]: This is for school admins. We won't be providing counts for district admins 👍

Comment (johnhuangclever) [reply full-v2.yml]: Oh actually you're right, count will be supported for district and district admin

**Comment by meliaj** [comment]
LGTM assuming the count param situation matches reality

**Pull request merged**